charts/events/templates/deployment.yaml

```diff
@@ -73,7 +73,8 @@ spec:
             httpGet:
               path: /healthz
               port: 5171
-            timeoutSeconds: 5
-            failureThreshold: 10
+            initialDelaySeconds: 30
```

Comment:
I suspect this is unnecessary. This will delay the initial probe, which means that pods will never be ready within this interval. The observe-events pod would never be ready within 30 seconds even for small clusters. I think increasing `failureThreshold` and / or `periodSeconds` achieves the intended result without impacting smaller clusters.